### DISCUSSION TITLE
More contributors needed

### DISCUSSION THREAD

**Discussion by jschlyter** [pypolestar/polestar_api#214] [Announcements]
We need more eyes on the code, both for code review, debugging and new features. @leeyuentuen is still great help but as he no longer has a Polestar there's some things he cannot test. If you know your way around Python and Home Assistant, please get in touch!

**Comment by BJReplay** [comment]
Count me in - if you have no other better qualified takers.  Not yet a Polestar owner (mid-December ETA), but I'm learning Python 😉 have finally got myself a proper HA core Dev Container set up and maintain https://github.com/BJReplay/ha-solcast-solar (with a lot of help) and am in the process of getting another integration up and running (which is why I'm learning python).

I just added the integration to my Dev Container, hooked up my VIN for my newly ordered polestar 3, and could spot the factory completion date, so I'm looking forward to being a user.

I can test, now have a test rig up and running, so can assist there on PS3 at the very least and can rapidly git pull PR branches and test if nothing else.

**Comment by jschlyter** [comment]
I've added @wouterhardeman to the review team (cc @pypolestar/core) for now. Feel free to contribute with code reviews @BJReplay, the more eyes the better.

**Comment by ValtteriAho** [comment]
Hey i can help as i have Polestar 4.  I can test whenever is needed :)

**Comment by henkha** [comment]
I own a MY 2024 Polestar 2 since last week. Count me in when it comes to testing!

Thank you!

**Comment by umbex** [comment]
Hello, just installed with Polestar 4 and seems working fine! 
Happy to help in testing.

**Comment by marcolivierarsenault** [comment]
Polestar 3 here, happy to help and/or review. 


I have also build some Home assitant instegrations https://github.com/marcolivierarsenault/moonraker-home-assistant
